conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureContexts.java

```diff
@@ -68,6 +76,11 @@ public RequestContext createContext(HttpServerExchange exchange, Endpoint _endpo
             return cachedQueryParamsSnapshot;
         }
 
+        @Override
+        public void requestLogParameter(Arg<?> arg) {
```

Comment:
Ideas for a better method name?

Reply:
Yeah, was going to say: I think we discussed that I personally wouldn't be opposed if those actually showed up in normal logs as MDC as well, to aid debugging. So maybe #requestParameter is more generic (because it doesn't tie you to request LOGs)? We kinda used RequestContext already to name the whole concept here, so context is probably out of question.

Other ideas: #diagnosticParameter, but not too keen.

Reply:
`requestParameter` is nice, however it implies that the input is a parameter of the request, when it might come from elsewhere. I think the name you propose is easier to understand at a glance though.

Reply:
Maybe `requestArg(Arg<?>)` based on the `Arg` type name, avoiding naming collision with request parameters entirely, which I took from `requestLogV2.params`?